kong/clustering/init.lua

```diff
@@ -44,7 +46,16 @@ function _M.new(conf)
 end
 
 
+function _M:handle_cp_websocket()
+  return self.json_handler:handle_cp_websocket()
+end
+
+
 function _M:handle_wrpc_websocket()
+  if ngx.req.get_method() == "HEAD" then -- force fallback
```

Comment:
Any reference for me to understand this change?

Reply:
`HEAD` for endpoint `v1/wrpc` will return 404, which will disable 3.0 DP's wrpc